dotCMS/src/main/java/com/dotcms/rest/api/v1/experiments/ExperimentsResource.java

```diff
@@ -500,6 +518,45 @@ public ResponseEntityExperimentResults getResult(@Context final HttpServletReque
         return new ResponseEntityExperimentResults(experimentResults);
     }
 
+    /**
+     * Healthcheck for the Experiments/Analytics configuration.
+     *
+     */
+    @GET
+    @NoCache
+    @Path("/health")
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public ResponseEntityView healthcheck(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response)
+            throws DotDataException, DotSecurityException, SystemException, PortalException {
+
+        final InitDataObject initData = getInitData(request, response);
+        final Host host = WebAPILocator.getHostWebAPI().getCurrentHost(request);
+        final AnalyticsApp analyticsApp = Try.of(()->AnalyticsHelper.get().appFromHost(host))
+                .getOrNull();
+
+        if(analyticsApp==null) {
+            return new ResponseEntityView<>(Map.of("healthy", false));
+        }
+
+        final String url = analyticsApp.getAnalyticsProperties().analyticsWriteUrl();
+
+        final CircuitBreakerUrlBuilder builder = CircuitBreakerUrl.builder()
+                .setMethod(Method.POST)
+                .setUrl(url)
+                .setParams(map("token", analyticsApp.getAnalyticsProperties().analyticsKey()))
+                .setTimeout(4000)
+                .setHeaders(POSTING_HEADERS);
+
+        final Map<String, Object> testObject = Map.of("test", "test");
+
+        final EventsPayload eventPayload = new EventsPayload(testObject);
+        final Optional<Response> eventResponse = new EventLogRunnable(host, eventPayload)
+                .sendEvent(builder, new EventPayload(new JSONObject(testObject)));
+
+        return new ResponseEntityView<>(Map.of("healthy", eventResponse.isPresent()));
+    }
```

Comment:
I don't know if we should judt send a TRUE|FALSE here, maybe we should send a JSON Object, something like:


{
     healthy: true|false,
     [any attribute name to say that is possible get the token]:  true|false,
    connection: {
       jitsu: true| false,
       cubejs: false
    }
}

Reply:
I think for now it is ok like this so the UI can tell if it should show the message or not. We can then enhance for v2